Unique.thy

```diff
@@ -0,0 +1,35 @@
+(********
+Isabelle/HoTT
+Apr 2019
+
+Uniqueness in Unit type
+********)
+
+theory Unique
```

Comment:
Small collections of lemmas about particular types should usually go in the files where the types are defined.